<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1216</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab_planets">
       <attribute name="title">
        <string>Planets</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>CGalaxy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_CGalaxy"/>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>CSector</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_CSector"/>
          </item>
         </layout>
        </item>
        <item row="0" column="2" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>CSolarSystem</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_CSolarSystem"/>
          </item>
         </layout>
        </item>
        <item row="0" column="3" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>CPlanet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_CPlanet"/>
          </item>
         </layout>
        </item>
        <item row="0" column="4" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_Info"/>
          </item>
         </layout>
        </item>
        <item row="0" column="5">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="decimals">
             <number>10</number>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Generate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_ships">
       <attribute name="title">
        <string>Ships</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>681</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>№</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>From</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>To</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time</string>
         </property>
        </column>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>230</y>
          <width>551</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Отправка</string>
        </property>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>131</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QSpinBox" name="spinBox_3">
         <property name="geometry">
          <rect>
           <x>260</x>
           <y>20</y>
           <width>42</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_temp">
       <attribute name="title">
        <string>Temp</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_find">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>651</width>
          <height>181</height>
         </rect>
        </property>
        <property name="title">
         <string>Find</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lineEdit_find"/>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_findByName">
           <property name="text">
            <string>Find by name</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinBox_find"/>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_findByID">
           <property name="text">
            <string>Find by ID</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>211</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QLabel" name="label_find">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Visualization</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" rowspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_draw">
             <property name="text">
              <string>Draw</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_items">
             <property name="text">
              <string>Items</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>419</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1216</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
